pg_search/sql/pg_search--0.7.1--0.7.2.sql

```diff
@@ -0,0 +1 @@
+\echo Use "ALTER EXTENSION pg_search UPDATE TO '0.7.2'" to load this file. \quit
```

Comment:
We changed `_bootstrap.sql` so those changes need to be applied here.

Reply:
See #1185